packages/actions/index.ts

```diff
@@ -779,6 +779,11 @@ export const setWelcomeComplete = (state) => {
   return setState(payload)
 }
 
+export const verifyPrivacyPolicy = (state) => {
+  const payload = { privacyPolicyVerified: state }
+  return setState(payload)
+}
+
```

Comment:
I don't know enough about how RN saves state, so I have to ask: does this save to native user defaults? 
Does it save to disk and survive across App reboots?

Reply:
this will not survive across app reboots, but it won't matter because this screen will only show up on the create account flow. if the user has already made an account then this screen will not show up